test/capi/capi_utils.hpp

```diff
@@ -197,6 +197,36 @@ static tc_sarray* make_sarray_double(const std::vector<double>& v) {
   return sa;
 }
 
+static tc_sarray* make_sarray_integer(const std::vector<int64_t>& v) {
```

Comment:
You may need `__attribute__((__unused__))` if this header is included in any test files that don't call this function -- otherwise you may get build errors on various platforms.

Reply:
Done. Thank you so much for the note!